### DISCUSSION TITLE
Handwritten Sticky-Notes?

### DISCUSSION THREAD

**Discussion by ser1us** [hadynz/obsidian-kindle-plugin#299] [Q&A]
Greetings,

I am unable to find anything in the issues or in the discussions about this. I own a Scribe and have just installed the plugin. Everything is syncinc nicely. Except for handwritten sticky notes.

Is there a way you can download them too? Maybe something I need to enable somewhere?

Thank you very much for this plugin, already a fantastic addition in my obsidian workflow.

Cheers

**Comment by donbarger** [comment]
That would be great! I hope someone figures out how to do that!
